assets/src/editor/events/event-editor.js

```diff
@@ -9,7 +9,12 @@ const container = document.getElementById( 'normal-sortables' );
 const editor = document.createElement( 'div' );
 editor.className = 'ee-editor-div';
 container.prepend( editor );
-const eventId = window.eeEditorEventId;
+const eventData = window.eeEditorEventData;
+console.log( '%c > eventData: ', 'color: #BCBDAC;', eventData );
+const eventId = eventData.event && eventData.event.EVT_ID ?
+	eventData.event.EVT_ID :
+	0;
+console.log( '%c > eventId: ', 'color: #BCBDAC;', eventId );
```

Comment:
don't worry about these console logs, they are so Manzoor can see how data is structured